clang/lib/AST/PrintfFormatString.cpp

```diff
@@ -783,6 +783,9 @@ bool PrintfSpecifier::fixType(QualType QT, const LangOptions &LangOpt,
   case BuiltinType::SatUShortFract:
   case BuiltinType::SatUFract:
   case BuiltinType::SatULongFract:
+  case BuiltinType::DecimalFloat32:
+  case BuiltinType::DecimalFloat64:
+  case BuiltinType::DecimalFloat128:
```

Comment:
This might be something to review in the future. The `printf()` length modifiers for `_Decimal32`, `_Decimal64`, and `_Decimal128` are `H`, `D`, and `DD`.

Reply:
I filed issue #19 to follow up on this.